CONTRIBUTING.md

```diff
@@ -1,37 +1,82 @@
-## How to contribute to Coraza WAF
+# Contributing to Coraza
 
-#### **Did you find a bug?**
+>**Note**: We take Coraza's security and the trust of our community seriously. If
+> you believe you have found a security issue in Coraza or any of its
+> components, please responsibly disclose by contacting us. See
+> [SECURITY.md](https://github.com/corazawaf/coraza/blob/v2/master/SECURITY.md)
+> for details.
 
-* **Do not open up a GitHub issue if the bug is a security vulnerability
-  in Coraza**, and instead to refer to our [security policy](https://github.com/corazawaf/coraza/blob/v2/master/SECURITY.md).
 
-* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/corazawaf/coraza/issues).
+We are striving to support an open community for the Coraza Project. We support
+our contributors, please don't feel afraid or unsure of submitting feedback or asking a question.
 
-* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/corazawaf/coraza/issues/new). Be sure to include a **title and clear description**, as much relevant information as possible, and a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+## Contributions
 
-#### **Did you write a patch that fixes a bug?**
+* Get in touch via the [OWASP Slack Community](https://owasp.org/slack/invite) (#coraza)
+* Provide feedback and report potential bugs
+* Suggest enhancements to the project
+* Perform tests and increase test coverage
+* Fix a [Bug](https://github.com/corazawaf/coraza/issues?q=is%3Aopen+is%3Aissue+label%3Abug) or implement an [Enhancement](https://github.com/corazawaf/coraza/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement)
+* Improve our Documentation, the [Coraza Website Repo](https://github.com/corazawaf/coraza.io) is on Github.
 
-* Open a new GitHub pull request with the patch.
+## Reporting an Issue
 
-* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+* Security related issues are covered by the [Security Policy](https://github.com/corazawaf/coraza/blob/v2/master/SECURITY.md)
+* Make sure you test against the latest version, it's possible the issue was
+  already fixed. However if you are on an older version of Coraza and feel the
+  issue is critical, please let us know.
+* Check existing [Issues](https://github.com/corazawaf/coraza/issues) (open and closed) to ensure it was not already reported.
+* Provide a detailed description and a reproducible test case in a new [Issue](https://github.com/corazawaf/coraza/issues/new).
+  Be sure to include as much relevant information as possible, a **code sample** or an **test case** demonstrating the fault helps us to reproduce your problem.
 
-* Before submitting, please commit using [pre-commit](https://pre-commit.com/) to validate your PR passes the quality tests
```

Comment:
We use the [pre-commit](https://pre-commit.com/) framework to perform quality tests before sending them for review. It is recommended you install it and use it before sending any patches.